### ISSUE TITLE
[Bug] Making new marker ending in space causes level to complete

### ISSUE THREAD

**Issue by 777arc** [tobspr/shapez.io#1309]
In the web version (cloned/built off master branch a week ago), it appears that making a new marker, with name ending in a space, e.g. "test marker ", causes the game to think the level just completed.  I don't know if there are any other triggers required, but I was on level 24 when I made the new marker, moved to 25 without building a single correct shape, and then once I figured out how to reproduce the bug I moved to level 26.

**Comment by 777arc** [comment]
Oh I'm not sure if this is required but the new marker ending in space had the same name as an existing marker.

**Comment by EmeraldBlock** [comment]
did you press `p` when typing the name of the marker?
(yes this sounds dumb but it's the most likely culprit)

**Comment by 777arc** [comment]
I don't think so, when I did the 2nd test I used "test " as the marker and it went to the next level again, I don't believe I hit p.

**Comment by EmeraldBlock** [comment]
sorry for a late response, but I can't seem to reproduce it.

some questions:
* could you provide a step-by-step of what you did?
* when you leveled up, was there a level-up screen or not?
* do you have any non-ordinary keybindings or anything?
* can you still reproduce it?

**Comment by 777arc** [comment]
There was a level up screen each time.  All default key bindings.  I essentially played like normal till level 24, it was my first time playing, and I started to use markers around that time, I only added the space at the end because I wanted a marker named something that already existed, I was going to delete the other marker after but it wouldn't let me name them the same so I added a space to the end, then immediately it said I leveled up.  I started playing with different marker names and figured out if I end it in a space it levels up, I did it two more times before moving on.  Not currently in a position to retest if it still bugs out unfortunately.
